docs/source/usage/picmi/intro.rst

```diff
@@ -113,13 +113,31 @@ Parameters/Methods prefixed with ``picongpu_`` are PIConGPU-exclusive.
 
 - **Simulation**
 
-  - ``__init__(..., picongpu_template_dir)``:
+  additional constructor/configuration options:
+  - ``picongpu_template_dir``:
```

Comment:
Doesn't render correctly. This happens multiple times down the site. I won't tag all of them.

Reply:
missing empty line, fixed